File: C/2_phep_toan_va_thu_vien/bai_10.c
/*
Create on Sat Sep 27 2025 15:15
@author: ntdai

#11 EP KIEU DU LIEU
	Cu phap: (type)value;
	Loai 1: Mo rong (short -> int -> long -> float -> double)
	Loai 2: Thu hep (double -> float -> long -> int -> short)
*/

#include "stdio.h"

int main(){
	int a, b;
	printf("Nhap a = ");
	scanf("%d", &a);
	printf("Nhap b = ");
	scanf("%d", &b);
	
	float kq_1 = (float)a/b;
	int kq_2 = (int)kq_1;
	
	printf("%d/%d=%.2f\n", a, b, kq_1);
	printf("kq_2 = %d", kq_2);
}

File: python/part_1/bai_10.py
'''
Create Sat Sep 27 2025 9:22
@author: ntdai

#10 TOÁN TỬ SO SÁNH VÀ LOGIC
'''
x = int(input("x: "))
y = int(input("y: "))
# True or False
print("{0} < {1} là {2}".format(x, y, x<y))
print("{0} > {1} là {2}".format(x, y, x>y))
print("{0} == {1} là {2}".format(x, y, x==y))
print("{0} != {1} là {2}".format(x, y, x!=y))
print("{0} <= {1} là {2}".format(x, y, x<=y))
print("{0} >= {1} là {2}".format(x, y, x>=y))
#--------------------------------------------
# Toán tử Logic
z = int(input("z: "))
print("({0}<{1}) and ({2}<{3}) = {4}".format(x, y, y, z, (x<y) and (y<z)))
print("({0}<{1}) or ({2}<{3}) = {4}".format(x, y, y, z, (x<y) or (y<z)))
print("not ({0}<{1}) = {2}".format(x, z, not (x<z)))
